book/src/bridge/attributes.md

```diff
@@ -36,4 +36,9 @@ For [`#[qproperty]`](./extern_rustqt.md#properties), a CXX or Rust name can be p
 
 > **⚠️ Deprecation warning**:
 > CXX-Qt <0.6 did automatic case conversion if no `#[cxx_name = "..."]` or `#[rust_name = "..."]` is specified.
-> Starting with CXX-Qt 0.7, this is no longer the case!
+> Starting with CXX-Qt 0.7, this is no longer the case! Automatic case conversion will be opt-in instead.
+
+### Automatic case conversion
+The `#[auto_cxx_name]` and `#[auto_rust_name]` attributes can be used to automatically rename cxx and rust names.
+These are placed at a block level on extern blocks, and will automatically case convert the items inside, unless they specify either rust or cxx name.
```

Comment:
might need to note it only works on `extern "C++Qt"` / `extern "RustQt"` blocks?

Reply:
Would move the part about it only working on `extern "C++Qt"` and `extern "RustQt"` blocks to this text